Resources/Prototypes/Entities/Objects/Tools/squeezebottle.yml

```
@@ -0,0 +1,129 @@
+- type: entity
+  parent: DrinkBase
+  id: SqueezeBottle
+  name: squeeze bottle
+  description:
+  components:
+  - type: ApplyReagentToItem
+    amountConsumedOnUse: 5
+    onSqueezeNoise: /Audio/Items/squeezebottle.ogg
+  - type: Openable
+    closeable: true
+    sound:
+      collection: packetOpenSounds
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/squeeze_bottle.rsi
+    layers:
+      - state: body
+      - state: cap_closed_red
+        map: ["enum.OpenableVisuals.Layer"]
+      - state: fill1
+        map: [ "enum.SolutionContainerLayers.Fill" ]
+        visible: false
+      - state: front
+        map: [ "enum.SolutionContainerLayers.Overlay" ]
```

```suggestion
    - state: body
    - state: cap_closed_red
      map: ["enum.OpenableVisuals.Layer"]
    - state: fill1
      map: [ "enum.SolutionContainerLayers.Fill" ]
      visible: false
    - state: front
      map: [ "enum.SolutionContainerLayers.Overlay" ]
```

There's a lot of inconsistency in this file with the list indentation. Usually it should either be `-` on the same indent as the `something:` or one deeper. I prefer on the same indentation, but for existing files follow whatever it has.

Above `components:` is on the same indent. `layers:` is not. Keep it consistent.

---

OK I think I fixed all the inconsistencies!